unicodetools/org/unicode/tools/Ids.java

```diff
@@ -984,7 +984,7 @@ private static void showRadicalCompare() throws IOException {
 
     private static void showRadicalCompareTxt() throws IOException {
 
-        try (PrintWriter out = FileUtilities.openUTF8Writer(orgSettings.Output.GEN_DIR + "ids/", "radicalCompare.txt");
+        try (PrintWriter out = FileUtilities.openUTF8Writer(Settings.Output.GEN_DIR + "ids/", "radicalCompare.txt");
```

Comment:
Apparently before I pushed my previous environment variable refactoring I fat-fingered "org" into the wrong window... oops!